src/Service.Tests/GraphQLRequestExecutor.cs

```diff
@@ -61,5 +72,63 @@ internal static class GraphQLRequestExecutor
 
             return graphQLResult.GetProperty("data").GetProperty(queryName);
         }
+
+        /// <summary>
+        /// Executes a graphql query and returns the result as a json element.
+        /// Used for multiple datasources tests.
+        /// </summary>
+        /// <param name="client">client</param>
+        /// <param name="configProvider">configProvider</param>
+        /// <param name="query">query</param>
+        /// <param name="variables">variables</param>
+        /// <param name="authToken">authToken.</param>
+        /// <param name="clientRoleHeader">clientRoleHeader</param>
+        /// <returns>JsonResult</returns>
+        public static async Task<JsonElement> PostGraphQLRequestAsync(
```

Comment:
I didn't see any tests that test:
1. issuing a GraphQL request to retrieve data from a multi data source scenario?
2. Where is this function PostGraphQLRequestAsync used in multi db scenario tests?

Reply:
Added test. The postGraphQLRequestAsync is not being used.